### PR TITLE
Add GigSalad (partially remote company)

### PR DESCRIPTION
**Pull request by johnbacon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ basho | http://basho.com/ | |
 BeBanjo | http://bebanjo.com/ | |
 Bizink | http://bizinkonline.com | |
 Black Pixel | https://blackpixel.com/ | |
-Bloc | https://bloc.io/ | |
+Bloc | https://bloc.io/ | |gi
```
Comment (jessicard) [inline README.md]: Uh oh! Typo I think :)

**Comment by jessicard** [comment]
@johnbacon Thanks for adding GigSalad! :) There are some merge conflicts - do you mind resolving? It would also be awesome if you could please fill out a company profile and link to it.

**Comment by johnbacon** [comment]
Apologies on the typo – ack!. Will resolve, create a company profile, and re-submit.

**Comment by jessicard** [comment]
Thank you so much for taking the time to do that!

**Comment by johnbacon** [comment]
Should be good to go! Let me know if I'm missing anything.

**Comment by jessicard** [comment]
Awesome! Thanks! :sparkles:

**Pull request merged**